hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -471,6 +473,30 @@
             job-env: |
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
                 export PROJECT="kubernetes-ingress"
+                # TODO: Enable this when we've split 1.2 tests into another project.
+                export FAIL_ON_GCP_RESOURCE_LEAK="false"
+        - 'gke-ingress-release-1.2':
```

Comment:
This needs to go in the gke-1.2 yaml project, or else it will still pull master binaries.